sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -216,7 +223,20 @@ export class EventProcessor {
             partitionId: partitionId
           };
 
-          const checkpointManager = new CheckpointManager();
+          for (const ownership of ownerships) {
+            if (ownership.partitionId != partitionId) {
+              const partitionOwnership: PartitionOwnership = {
+                eventHubName: this._eventHubClient.eventHubName,
+                consumerGroupName: this._consumerGroupName,
+                instanceId: generate_uuid(),
+                partitionId: partitionId,
+                ownerLevel: 0
+              };
+              await this._partitionManager.claimOwnerships([partitionOwnership]);
```

Comment:
Here, we are depending on the output of `listOwnerships` in order to claim ownership.
When Event Processor is first started for an event hub instance, `listOwnerships` will return empty array and this code will fail to claim ownership to any partition

Reply:
Ohh yes.. then I'll just remove `listOwnership()` method and claim ownership for each partition